homework-submission/week1/app.js

```diff
@@ -0,0 +1,85 @@
+// Create a todo list item as reusable component:
```

Comment:
The description of this component (and which props it accepts) could be a bit more elaborate in my opinion

Reply:
I wrote a detailed comment  in the changed file(2. comment)

Reply:
(2.commit)